manifests/repo/debian.pp

```diff
@@ -17,85 +17,25 @@
 # [*key*]
 #   Public key in apt::key format
 #
-# [*dotdeb*]
-#   Enable special dotdeb handling
-#
-# [*sury*]
-#   Enable special sury handling
-#
 class php::repo::debian(
-  $location     = 'http://packages.dotdeb.org',
-  $release      = 'wheezy-php56',
-  $repos        = 'all',
-  $include_src  = false,
-  $key          = {
-    'id'     => '6572BBEF1B5FF28B28B706837E3F070089DF5277',
-    'source' => 'http://www.dotdeb.org/dotdeb.gpg',
-  },
-  $dotdeb       = true,
-  $sury         = true,
```

Comment:
we should introduce a enum parameter like `repo_management` `auto|native|sury|(dotdeb)` so the user can decide how the repo management behaves. it should be possible to set a custom repo and prevent the auto added repo from below

Reply:
Parameters like `location`, `key` are configurable via hiera. With this PR all standard deb repo like sury are covered by this change. I do not see a point for sury special handing.